custom/icds_reports/static/icds_reports/mobile/css/map-chart.less

```diff
@@ -9,6 +9,7 @@
   .p-3;
   .rounded-border;
   max-height: ~"calc(100vh - 170px)";
+  height: ~"calc(100vh - 170px)";
```

Comment:
why was this necessary? and is it now redundant with `max-height`?

Reply:
height in styling defines height of the element when its child element height is less or equal to the parent height. If child element height goes beyond parent height, parent height matches with child height. So max height is set to prevent parent height from increasing.

Reply:
![Screenshot from 2020-01-17 15-02-01](https://user-images.githubusercontent.com/56248457/72601355-3ef02280-393b-11ea-9a95-d0499b2724ae.png)
This is how it looks when clicked on 'i'. Saket mentioned to keep it the way it is in web

Reply:
![Screenshot from 2020-01-17 15-13-40](https://user-images.githubusercontent.com/56248457/72601727-f8e78e80-393b-11ea-9394-f4271120189b.png)
If height is removed and when there is no data. it looks this way

Reply:
I see. So it's kind of an unrelated change? 

I actually prefer the section only growing to take the needed space. Perhaps if the issue is clicking the "i" with no data, you could just set a `min-height`?

Reply:
sure.